src/js/hooks/cycleCount/useToResolveTab.jsx

```diff
@@ -306,7 +306,7 @@ const useToResolveTab = ({
       ),
       cell: ({ getValue }) => (
         <TableCell className="rt-td">
-          {getValue()}
+          {getValue().toString()}
```

Comment:
what's the purpose of that? Is it some additional change or something related to deleting, which I doubt? :thinking:

Reply:
It is an additional request - when the value was 0, it wasn't visible